.github/workflows/push-test.yml

```diff
@@ -0,0 +1,17 @@
+name: Push Test
```

Comment:
I recommend giving this a meaningful name.  For example, in my current project, each job has a descriptive name such as API unit tests (the backend), Web unit tests (the frontend), Web Cypress tests etc.

Reply:
-Fixed, hopefully. Changed the name to Unit Tests. Right now since these are the only tests, I figured I could rename this job to be more specific later if I need to make a distinction.